src/App.js

```diff
@@ -1,12 +1,22 @@
 import "./App.css";
 import { Home } from "./pages/home/Home";
-import { Header } from "./shared/components/header/Header";
-
+import { HeroSection } from "./pages/home/components/heroSection/HeroSection";
+import { Routes, Route } from "react-router-dom";
+import { Explore } from "./pages/explore/explore";
+import { WithHeader } from "./shared/components/withHeader/WithHeader";
+import Mockman from "./mockMan";
 function App() {
   return (
     <div className="App">
-      <Header/>
-      <Home />
+      <Routes>
+        <Route path="/" element={<WithHeader />}>
+          <Route path="mockApi" element={<Mockman />}></Route>
+          <Route path="/" element={<Home />}>
+            <Route index element={<HeroSection />} />
+            <Route path="explore" element={<Explore />} />
```

Comment:
Why explore page path is under home route??

Reply:
Because I am using nested route and inside home component I am using outlet .